### PR TITLE
upgrade to hammer-cli-katello 0.0.24

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
should fix the pipeline

File: rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 1]
```diff
@@ -72,6 +72,7 @@ cp -pa .%{gem_dir}/* \
 %files doc
 %doc %{gem_docdir}
 %doc %{gem_instdir}/config
+%doc %{gem_instdir}/test
```
Comment (ehelms) [inline rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec:75]: We don't want to just exclude this?

Comment (jlsherrill) [inline rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec:75]: i was just doing what hammer_cli_foreman did

Comment (ehelms) [inline rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec:75]: okie doke

**Comment by ehelms** [comment]
ACK

**Pull request merged**